torch/csrc/jit/script/tree_views.h

```diff
@@ -543,7 +548,8 @@ struct UnaryOp : public Expr {
     }
   }
   static UnaryOp create(const SourceRange& range, int kind, const Expr& expr) {
-    return UnaryOp(Compound::create(kind, range, {expr}));
+    auto unary_kind = kind == '-' ? TK_UNARY_MINUS : kind;
```

Comment:
Uh what? Can we patch all the call sites instead? This is some really unexpected logic